templates/compose/soketi.yaml

```diff
@@ -0,0 +1,20 @@
+# documentation: https://docs.soketi.app
+# slogan: Soketi is your simple, fast, and resilient open-source WebSockets server.
+# tags: websockets,open,source,messaging
+# logo: svgs/soketi.png
```

Comment:
I recommend the new logo: https://avatars.githubusercontent.com/u/70626942?s=400&u=4b09731b5a6771c46cba09662109acd154cd2723&v=4